demo/src/main/java/com/eis/geoCalendar/demo/RemoveLocatedEventDialog.java

```diff
@@ -0,0 +1,63 @@
+package com.eis.geoCalendar.demo;
+
+import android.app.AlertDialog.Builder;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.DialogFragment;
+
+import com.google.android.gms.maps.model.Marker;
+
+/**
+ * @author Kumar
+ * <p>
+ * This class defines a DialogFragment to remove a Located Event mark
+ */
+public class RemoveLocatedEventDialog extends DialogFragment {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class should define a constructor.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4807675)

Reply:
Spe che tolgo sta roba